fvm/transactionEnv.go

```diff
@@ -179,6 +180,11 @@ func (e *TransactionEnv) GetAuthorizedAccountsForContractUpdates() []common.Addr
 	return addresses
 }
 
+func (e *TransactionEnv) UseContractAuditVoucher(address runtime.Address, code []byte) (bool, error) {
```

Comment:
Could be private.

Reply:
Good idea, addressed https://github.com/onflow/flow-go/pull/1826/commits/07ac4a829bfc31538a0825507fcecbf262f9ff91
Also updated the core-contracts branch now that it's merged